### PR TITLE
updates to README files, added CODE_OF_CONDUCT and CONTRIBUTING files

### PR DESCRIPTION
**Pull request by andogro**

### CODE REVIEW

File: bridge-nodejs/README.md [hunk 1]
```diff
@@ -22,44 +30,69 @@ Playbooks has two points of configuration - file `hosts.yml` and `group_vars/<br
...
+* `-i <file>` - use specified file as a `hosts.yml` file.
 
-`-i <file>` - use specified file as a `hosts.yml` file.
+* `-e "<variable>=<value>"` - override default variable.
 
-`-e "<variable>=<value>"` - override default variable.
+* `--private-key=<file_name>` - if private keyfile is required to connect to the ubuntu instance.
 
-Useful variables:
+**Useful variables:**
```
Comment (andogro) [inline bridge-nodejs/README.md]: Where would these be used - are they added to the hosts.yml file?

File: bridge-nodejs/README.md [hunk 2]
```diff
@@ -22,44 +30,69 @@ Playbooks has two points of configuration - file `hosts.yml` and `group_vars/<br
...
+
+The Bridge service is named `poabridge`. Use the default `SysVinit` commands to `start`, `stop`, `restart`, and `rebuild` the service and to check the `status` of the service. 
+
+Commands format:
 ```bash
 sudo service poabridge [start|stop|restart|status|rebuild]
 ```
 
-# Obtaining logs
+## Logs
```
Comment (andogro) [inline bridge-nodejs/README.md]: I'm currently getting an error when I run this command. Not sure if there's another step I'm missing:

```
ubuntu@ip-172-31-62-48:~/bridge$ docker-compose logs  ERROR: Couldn’t connect to Docker daemon at http+docker://localhost - is it running?
If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
```

File: bridge-nodejs/README.md [hunk 3]
```diff
@@ -22,44 +30,69 @@ Playbooks has two points of configuration - file `hosts.yml` and `group_vars/<br
...
+
+
+3. Create a file for the public bridge parameters. 
+   1. Go to the group_vars folder. 
+   `cd group_vars`
+   2. Copy the example file and rename with your <bridge_name>. For example, if the bridge name in hosts.yml is sokol-kovan, you would name the file sokol-kovan.yml
+   `cp example.yml <bridge_name>.yml` 
+
+
+`group_vars/<bridge_name>.yml` contains the public bridge parameters. `group_vars/example.yml` shows an example configuration for the POA/Sokol - POA/Sokol bridge. Parameter values should match values from the .env file for the token-bridge. See https://github.com/poanetwork/token-bridge#configuration-parameters for details.
```
Comment (andogro) [inline bridge-nodejs/README.md]: How do validators get the information for the `group_vars/<bridge_name>.yml` file? Will it typically be sent by an admin or will they have access to the token-bridge .env file?

File: bridge-nodejs/README.md (hunk 1 above)
Comment (phahulin) [reply bridge-nodejs/README.md]: Yes, they can be added to hosts.yml to override defaults.

File: bridge-nodejs/README.md (hunk 2 above)
Comment (phahulin) [reply bridge-nodejs/README.md]: I guess the docker itself didn't start, could you check with `docker ps -a` what's the container's status?

File: bridge-nodejs/README.md (hunk 3 above)
Comment (phahulin) [reply bridge-nodejs/README.md]: The idea is that group_vars/<bridge_name>.yml files are prepared by admins for each bridge and validator just use a required name for the bridge in hosts.yml to let ansible know which file to use

**Comment by phahulin** [comment]
> Do we want to change the name of the bridge-nodejs folder to token-bridge for consistency? 
Yes, I think we should do that but as the last step after all outstanding PRs are merged so that developers don't get weird errors

> upgradable-wo-parity
I agree (again, at the last step). to `rust-bridge`?

> Do we need a file similar to the DETAILS file
I think yes, it might be a good idea to explain more technical details for those who is interested.

File: bridge-nodejs/README.md [hunk 4]
```diff
@@ -1,65 +1,117 @@
...
 
 `hosts.yml` should have the following structure:
 
 ```yaml
 <bridge_name>:
     hosts:
         <host_ip>:
-            ansible_user: "<user>"
+            ansible_user: <user>
             VALIDATOR_ADDRESS: "<hex address>"
```
Comment (akolotov) [inline bridge-nodejs/README.md]: `VALIDATOR_ADDRESS` is not used anymore for this configuration file.

File: bridge-nodejs/README.md [hunk 5]
```diff
@@ -1,65 +1,117 @@
...
             #syslog_server_port: "<protocol>://<ip>:<port>" # When this parameter is set all bridge logs will be redirected to <ip>:<port> address.
 ```
 
-You can check `hosts.yml.example` as example. `hosts.yml` can contain multiple hosts and bridge configurations (groups) at once.
+| Value | Description |
+|------------------------------------------------|:----------------------------------------------------------------------------------------------------------:|
+| <bridge_name> | The bridge name which tells Ansible which file to use. This is located in `group_vars/<bridge_name>.yml`. |
+| <host_ip> | Remote server IP address. |
+| ansible_user: <user> | User that will ssh into the node. This is typically `ubuntu` or `root`. |
+| VALIDATOR_ADDRESS: "<hex address>" | The validator's address, beginning with the `0x` prefix. |
```
Comment (akolotov) [inline bridge-nodejs/README.md]: `VALIDATOR_ADDRESS` is not used anymore for this configuration file.

File: bridge-nodejs/README.md [hunk 6]
```diff
@@ -1,65 +1,117 @@
...
+
+
+3. Copy the bridge name(s) to the hosts.yml file. 
+   1. Go to the group_vars folder. 
+   `cd group_vars`
+   2. Note the  <bridge_name> and add it to the hosts.yml configuration. For example, if a bridge file is named sokol-kovan.yml, you would change the <bridge_name> value in hosts.yml to sokol-kovan.
+
+`group_vars/<bridge_name>.yml` contains the public bridge parameters. This file will be prepared by administrators for each bridge. The validator only needs to add the required bridge name in the hosts.yml file to tell Ansible which file to use.
+
+ `group_vars/example.yml` shows an example configuration for the POA/Sokol - POA/Sokol bridge. Parameter values should match values from the .env file for the token-bridge. See https://github.com/poanetwork/token-bridge#configuration-parameters for details.
```
Comment (akolotov) [inline bridge-nodejs/README.md]: I would also add that `roles/repo/tasks/main.yml` should be changed to specify specific branch or commit in the `token-bridge` repo. The parameter `versions` is used for this. Absence of this parameter means that the default branch (`master`) is used.

E.g. the content of the file below means that `support-erc20-native-#81` branch will be used for sources of the Token bridge.
```yml
- name: Get bridge repo
  git:
    repo: "{{ bridge_repo }}"
    dest: "{{ bridge_path }}"
    force: yes
    version: support-erc20-native-#81
```

File: bridge-nodejs/README.md (hunk 4 above)
Comment (andogro) [reply bridge-nodejs/README.md]: I deleted and also removed from the hosts.yml.example file

**Pull request merged**